tests/pkg/elk/main.c

```diff
@@ -1,19 +1,9 @@
 /*
- * Copyright (C) 2013-2021 Cesanta Software Limited
- * All rights reserved
- *
- * This software is dual-licensed: you can redistribute it and/or modify
- * it under the terms of the GNU General Public License version 2 as
- * published by the Free Software Foundation. For the terms of this
- * license, see <http://www.gnu.org/licenses/>.
- *
- * You are free to use this software under the terms of the GNU General
- * Public License, but WITHOUT ANY WARRANTY; without even the implied
- * warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
- * See the GNU General Public License for more details.
- *
- * Alternatively, you can license this software under a commercial
- * license, please contact us at https://cesanta.com/contact.html
+ * SPDX-FileCopyrightText: 2013-2021 Cesanta Software Limited
+ * SPDX-License-Identifier: GPL-2.0-only
+ * SPDX-FileCopyrightText: Alternatively, you can license this software under a
+ * SPDX-FileCopyrightText: commercial license, please contact us at
+ * SPDX_FileCopyrightText: https://cesanta.com/contact.html
```

Comment:
I shortened this quite a bit. This is probably the controversial part of this PR